Failed to compile project when importing redux-pack

Node version: v7.2.0
Project created via create-react-app with react-scripts v0.7.0

I've tried to install with both `npm install -S redux-pack` and `yarn add redux-pack`, but when I'm importing redux-pack into project:
```import { middleware as reduxPackMiddleware } from 'redux-pack';```

I'm receiving this error:
```
Failed to compile.

Error in ./~/redux-pack/lib/handle.js
Module not found: 'deline' in [path to project]/node_modules/redux-pack/lib

 @ ./~/redux-pack/lib/handle.js 1:1303-1320
```

I've already tried manually install 'deline' module and it appears in 'node_modules' folder, but haven't succeed.

Can someone please share any thoughts on possible solutions and I will try to fix it?

--- a/package.json
+++ b/package.json
@@ -29,11 +29,10 @@
   },
   "homepage": "https://github.com/lelandrichardson/redux-pack#readme",
   "peerDependencies": {
-    "redux": "*",
-    "normalizr": "*"
+    "redux": "*"
   },
   "dependencies": {
-    "deline": "^1.0.2",
+    "deline": "^1.0.4",
     "invariant": "^2.2.2",
     "uuid": "^3.0.1"
   },

